dev-lang/jimtcl/jimtcl-0.82.ebuild

```
@@ -0,0 +1,52 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+SRC_URI="https://github.com/msteveb/jimtcl/archive/${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~arm ~arm64 ~m68k ~mips ~s390 ~x86"
+
+DESCRIPTION="Small footprint implementation of Tcl programming language"
+HOMEPAGE="http://jim.tcl.tk/"
+
+LICENSE="LGPL-2"
+SLOT="0/${PV}"
+IUSE="doc static-libs"
+
+DEPEND="
+	dev-lang/tcl:0
+"
+
+src_prepare() {
+	# Avoid tests that need networking
+	rm tests/ssl.test || die
+	default
+}
+
+src_configure() {
+	# Use the provided bootstrap tclsh, avoiding the script picking a different one
+	$(tc-getCC) -o jimsh0 autosetup/jimsh0.c
```

I think you need a `|| die` here, otherwise you can also use `edo` (would need to inherit `edo` too) like
```suggestion
	edo $(tc-getCC) -o jimsh0 autosetup/jimsh0.c
```